config/config_env.rb.example

```diff
@@ -0,0 +1,11 @@
+# Copy this file to [app]/config/congig_env.rb
+
+config_env :development, :test do
+  set 'API_HOST', 'http://localhost:9292/api/v1/'
```

Comment:
Okay, I'll make a commit to change the example to 9393. The API runs on 9292, so just in case someone forgets in the future and is stuck wondering why there's a conflict